### PR TITLE
📚 added input examples

### PR DESCRIPTION
**Pull request by dmiller15**
# Pull Request Template

## Description

Updates readme with example inputs.

Resolves https://github.com/d3b-center/bixu-tracker/issues/660

## Type of change

- [x] Documentation

## How Has This Been Tested?

- [x] Simulated markdown for formatting

**Test Configuration**:
* Environment: Firefox 76.0.1
* Test files: readme.md

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -69,6 +69,193 @@ also be provided: `wxs_bait_interval_list` and `wxs_target_interval_list` for Hs
...
+      "class": "File",
+      "location": "/path/to/sample2_R2.fastq"
+    },
+    {
+      "class": "File",
+      "location": "/path/to/sample3_R2.fastq"
+    }
+  ],
+  "inputs_pe_rgs_list": [
+    "@RG\tID:sample1\tLB:blah\tPL:ILLUMINA\tSM:blah",
```
Comment (migbro) [inline readme.md:181]: Maybe make this slightly more helpful, like for LB could use a flow cell ID, and SM just a generic `sample_name` would do

Comment (dmiller15) [reply readme.md]: changed it to be more helpful.

File: readme.md [hunk 2]
```diff
@@ -69,6 +69,198 @@ also be provided: `wxs_bait_interval_list` and `wxs_target_interval_list` for Hs
...
+before performing steps like BQSR and Metrics collection.
+
+#### BAM Inputs
+The BAM processing portion of the pipeline is the simplest when it comes to inputs. You may provide
+a single BAM or many BAMs. The input for BAMs is a file list. In Cavatica or other GUI interfaces,
+simply select the files you wish to process. For command line interfaces such as cwltool, your input
+should look like the following.
+```json
+{
+  ...,
```
Comment (sickler-alex) [inline readme.md:90]: What do the three dots mean?

Comment (dmiller15) [reply readme.md:90]: Mainly just showing that the contents within the JSON block is part of a greater whole and should not be interpreted as a valid input on its own.

**Pull request merged**